### PR TITLE
docs: fix typo in readme

### PR DESCRIPTION
**Pull request by Eddie344**

### CODE REVIEW

**Review by danielroe — CHANGES_REQUESTED** [review]
Thanks! Would you mind reverting the lint/style/whitespace changes?

**Comment by Eddie344** [comment]
> Thanks! Would you mind reverting the lint/style/whitespace changes?

@danielroe  yes, sure

**Comment by Eddie344** [comment]
@danielroe done

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Alternative ports to check.
 
 ### `portRange`
 
-Alternative port range to check. Deefault is `[3000,3100]`
+Alternative port range to check. Default is `[3000,3100]`
```
Comment (danielroe) [inline README.md:71]:
```suggestion
Alternative port range to check. Default is `[3000, 3100]`
```

**Review by danielroe — APPROVED** [review]
Thank you! ❤️

**Pull request merged**